cli/script/command-executor.ts

```diff
@@ -178,6 +178,24 @@ function appList(command: cli.IAppListCommand): Promise<void> {
         });
 }
 
+function checkIfAppExists(appName: string): Promise<App> {
+    log(chalk.cyan(`Checking if app with the name "${appName}" exists...\n`));
+    return Promise<App>((resolve, reject) => {
+        sdk.getApps()
```

Comment:
You can actually call `sdk.getApp(<name>)` directly, there is no need to do this filtering logic on the client side as it is already being done on the server. And the promise will be rejected for you if it does not exist. Actually, given this, you could maybe remove this function completely, and just call `sdk.getApp(<name>)` within the `release` command, with a comment explaining why